### ISSUE TITLE
Init image help text may be slightly incorrect

### ISSUE THREAD

**Issue by halr9000** [rkhamilton/vqgan-clip-generator#73]
From [this line](https://github.com/rkhamilton/vqgan-clip-generator/blame/main/README.md#L177), where it says random noise will be used if no init image -- is that correct? This seems to conflict with lower down where the default for `init_noise` is `none`.

**Comment by rkhamilton** [comment]
I THINK that line in the docs is correct, though I can see the confusion. This was my attempt to understand what the original algorithm did and explain it.

If you look at [this else-block](https://github.com/rkhamilton/vqgan-clip-generator/blob/f0a2fb3e172e8d7ebbab848c7c834991f196d632/src/vqgan_clip/engine.py#L284) in engine.py you will see the code path followed when init_noise is none. It feeds [a randomly generated tensor](https://pytorch.org/docs/stable/generated/torch.randint.html) of appropriate dimensionality into a PyTorch routine [one_hot ](https://pytorch.org/docs/stable/generated/torch.nn.functional.one_hot.html) to generate a seed which gets iterated on by the subsequent algorithm. So, in short, it starts from a random tensor defined in a particular way.

If you don't use the default, and instead use "pixels" then the algorithm synthesizes an actual init_image from random pixels using [this function](https://github.com/rkhamilton/vqgan-clip-generator/blob/f0a2fb3e172e8d7ebbab848c7c834991f196d632/src/vqgan_clip/_functional.py#L49), and feeds that into the generator as if it were an init_image.

I haven't extensively tested both types of random noise initialization, and can't say why they are both in the codebase that I started from.

Let me know if that makes sense. If you can suggest a better way to explain the parameters I'd be happy to hear it.

**Comment by halr9000** [comment]
Cool, thanks for the lesson, and yes that makes sense. Only improvement I can think of would be to go back to init_noise in the table, and say something about the default (whether you call it none or not) is...still noise. :D 

I tried one vqgan project a few weeks back and it was really oddly inaccurate. I didn't find out the cause until I lowered save_every. Turned out it starts not with noise, but with a random imagenet image! And by random, I mean prompt could be "castle", and iteration 0-10 are clearly a dog or cat or etc. :D 

TY
